meilisearch-release-changelog/v1.14.0.md

```diff
@@ -0,0 +1,137 @@
+# v1.14.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+> [!WARNING]
+> Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports[#new]]((https://github.com/meilisearch/meilisearch/issues/new)/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.14 exposes a more granular way to express your filterable attributes! 🎉 This release also improves the performances of semantic search, the indexing speed of embeddings and a new route to get multiple documents by IDs.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Granular Filterable Attribute Settings
+
+
+This feature allows for the choice of the filter features of each filterable attribute with high granularity.
+Activating or deactivating a feature will have an impact on the indexing time.
```

Comment:
Can we make it more positive? looks like the feature will make the indexing time longer